docs/lab/ubuntu/ooto-driver.yaml

```
@@ -0,0 +1,48 @@
+apiVersion: kmm.sigs.k8s.io/v1beta1
+kind: Module
+metadata:
+  name: kmm-ci-a
+spec:
+  driverContainer:
+    container:
+      modprobe:
+        moduleName: kmm-ci-a 
+
+      kernelMappings:
+      #Matching kernel with this literal will trigger build unless same image and version exists.
+        - literal: 5.15.0-43-generic
+          containerImage: myimage.registry/kmmo-lab:1.0 
+          build:
+            dockerfile: |
+              ARG KERNEL_VERSION=''
+              FROM ubuntu as builder
+              RUN apt-get update && apt-get install -y bc \
+                  bison \
+                  flex \
+                  libelf-dev \
+                  gnupg \
+                  wget \
+                  git \
+                  make \
+                  bc \
+                  gcc \
+                  linux-headers-5.15.0-43-generic
```

Wouldn't it be interesting to do
```suggestion
                  linux-headers-${KERNEL_VERSION}
```
here and to make the mapping a `regexp` one?